adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -152,8 +167,27 @@ private void validateAcquireTokenRequest(final AuthenticationRequest authenticat
         }
 
         // validate authority
+        Telemetry.getInstance().startEvent(authenticationRequest.getTelemetryRequestId(),
```

Comment:
the idea was to have all the timing logic in one place. which does not make sense too much for aggregation but will make sense for default dispatcher.

---
In reply to: [90993840](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/736#discussion_r90993840) [](ancestors = 90993840)